src/DiffEqBase.jl

```diff
@@ -125,6 +125,7 @@ include("internal_euler.jl")
 include("init.jl")
 include("forwarddiff.jl")
 include("chainrules.jl")
+include("precompile.jl")
```

Comment:
```suggestion
if Base.VERSION >= v"1.4.2"
    include("precompile.jl")
end
```
if you want to be absolutely sure you're avoiding triggering a C-assertion (xref https://github.com/JuliaLang/julia/pull/35378).

Reply:
Just realized this library already requires v1.6